src/pages/trade/ui/order-form/segmented-control.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Direction } from './order-form-store';
+import cn from 'clsx';
+
+export const SegmentedControl: React.FC<{
+  direction: Direction;
+  setDirection: (direction: Direction) => void;
+}> = ({ direction, setDirection }) => {
+  return (
+    <div className='flex w-full h-8 mb-4'>
+      <button
+        className={cn(
+          'flex-1 border transition-colors duration-300 rounded-l-2xl focus:outline-none',
+          'border-r-0 border-other-tonalStroke',
+          direction === Direction.Buy
+            ? 'bg-success-main border-success-main text-text-primary'
+            : 'bg-transparent text-text-secondary',
+        )}
+        onClick={() => setDirection(Direction.Buy)}
+      >
+        Buy
```

Comment:
the buy button should remain disabled until an amount is entered and the estimate is calculated.